### ISSUE TITLE
for Retro_Person

### ISSUE THREAD

**Issue by LankyBox01** [LankyBox01/LankyBox01#2]
discuss here about the collab

**Comment by retro-person** [comment]
Hi, wassup?

**Comment by LankyBox01** [comment]
lol, great. lemme get the link to my yt

**Comment by LankyBox01** [comment]
https://www.youtube.com/channel/UCHwtT6K-PpdUzNWrE17kTXw
There. I will remove the link once you visit it for some privacy reasons

**Comment by LankyBox01** [comment]
*I also forgot to mention it's in romanian

**Comment by retro-person** [comment]
Ah, seen it. It's fine, I have realtime Google Translate for webpages in Chrome.

**Comment by LankyBox01** [comment]
Cool, so shall we write a script?

**Comment by retro-person** [comment]
Sure!

**Comment by LankyBox01** [comment]
Ok, let's start from the beginning - what's this story about?

**Comment by retro-person** [comment]
Hmm! I don't really know. Maybe we should pick a game first; my favorite is MineCraft.

**Comment by LankyBox01** [comment]
Nah, we need to use gacha life - maybe a guns game

**Comment by retro-person** [comment]
Oh, okay; Gacha Life is fine. Just a warning: I'm not a big fan of shooter games though; sorry. :~\

**Comment by LankyBox01** [comment]
So, my idea is that i am playing a so-called "game", which is actually just a gacha life animation. You just animate it and i put my facecam on it.

**Comment by LankyBox01** [comment]
i can also edit it

**Comment by retro-person** [comment]
Ah, cool!

**Comment by LankyBox01** [comment]
give me a bunch of ideas

**Comment by LankyBox01** [comment]
so i can write a script

**Comment by retro-person** [comment]
Hmm, okay. Ummmm... How about you're chasing a robber or something?

**Comment by retro-person** [comment]
> Hmm, okay. Ummmm... How about you're chasing a robber or something?

I mean, the "player" will be chasing the "robber".

**Comment by LankyBox01** [comment]
Love that! Maybe i am running with a.. sword i guess.
I am running with a sword to attack a player that has no weapon

**Comment by retro-person** [comment]
Perfect! Gacha Life has a sword prop.

**Comment by LankyBox01** [comment]
I know, that's why i chose it. I've used gacha life before :)

**Comment by retro-person** [comment]
Oh you did? Wow! i never thought anyone used gacha life except me :~P

**Comment by LankyBox01** [comment]
lol

**Comment by LankyBox01** [comment]
Let's call the player (me) as "Main" and the player with no weapon as "NoW".
So Main will Jump on NoW, and Main succeds to catch NoW and kill him. Now what?

**Comment by retro-person** [comment]
Good idea! Sorry I won't be able to answer for a while; got to go do homework and wait for my Lego `<deepvoice>` **MINDSTORMS** `</deepvoice>` RCX from 1998 to arrive. :~P

**Comment by LankyBox01** [comment]
lol, i will animate some of the parts to save you some work, have fun. We'll talk tomorrow!

**Comment by retro-person** [comment]
Okay; I'm back (got a quick homework break). When you're done animating, can you make an unlisted video of the animation and give me the link?

**Comment by LankyBox01** [comment]
In my country it's currently 1:50 AM, can we do this tomorrow lol

I am on my phone rn

**Comment by LankyBox01** [comment]
Ok, i am back.

**Comment by LankyBox01** [comment]
Here's the vid. I just need to add my facecam and stuff
[Video GachaLife.zip](https://github.com/LankyBox01/LankyBox01/files/5635089/Video.GachaLife.zip)
